docs/recipes/recipe-data-structures-in-git/index.md

```diff
@@ -0,0 +1,366 @@
+
+On top of that, detailed change history can be crucial for such large organizations. It is important to be able to tell in fine detail what was changed, when, and by whom.
+
+The Snowplow Console's UI offers excellent facilities to get started quickly with Data Structures (either using the Builder or directly the JSON editor), and is a solid tool for smaller teams; but it doesn't implement such approval processes, neither does it offer such fine-grained visibility around changes.
+
+A common solution when faced with these requirements is to move management to some form of version control platform (github/gitlab). This opens up an entire ecosystem of tools and patterns enabling all manner of custom workflows.
+
+We have built [snowplow-cli](/docs/understanding-tracking-design/managing-your-data-structures/cli/) to help you bridge the gap between these repository based workflows and BDP Console.
+
+## Prerequisites
+
+* A deployed Snowplow BDP pipeline
+* [snowplow-cli](/docs/understanding-tracking-design/managing-your-data-structures/cli/#download) downloaded and configured
+* A familiarity with [git](https://git-scm.com/) and an understanding of [github actions](https://docs.github.com/en/actions/writing-workflows)
+* A sensible [terminal emulator](https://en.wikipedia.org/wiki/Terminal_emulator) and shell
+
+
+## What you'll be doing
+
+This recipe will walk through creating and deploying a data structure from the command line using [snowplow-cli](https://github.com/snowplow-product/snowplow-cli). It will then show how it is possible to automate the validation and deployment process using [github actions](https://docs.github.com/en/actions/writing-workflows).
+
+## Create a local data structure
+
+Firstly we'll need a place to put things.
+
+```bash
+$ mkdir -p snowplow-structures/data-structures
+$ cd snowplow-structures
+```
+:::tip
+snowplow-cli data structures commands default to looking for data structures in `./data-structures`.
+:::
+
+Now let's create our data structure. We'll create a custom event called 'login'.
+
+```bash
+$ snowplow-cli ds generate login --vendor com.example
+```
+:::note
+`ds` is an alias for `data-structures`.
+:::
+
+This should provide us the following output
+
+```
+3:00PM INFO generate wrote=data-structures/com.example/login.yaml
+```
+
+You'll note that snowplow-cli uses a naming scheme that follows a `vendor/name` pattern and defaults to `yaml` output. This is not a prerequisite for other commands to work, just an opinionated default.
```

Comment:
This paragraph is unclear to me.

Reply:
I was trying to get across the pattern but also that you don't have to follow it. Maybe that isn't important to note. I'll reword